XPlot.DotNet.Interactive.KernelExtensions/PlotlyChartExtensions.cs

```diff
@@ -1,32 +1,28 @@
 ﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
+using System;
 using System.Text;
-using HtmlAgilityPack;
+using Microsoft.DotNet.Interactive.Formatting;
 using XPlot.Plotly;
 
 namespace XPlot.DotNet.Interactive.KernelExtensions
 {
     public static class PlotlyChartExtensions
     {
-        public static string GetChartHtml(this PlotlyChart chart)
+        public static string GetHtml(this PlotlyChart chart)
         {
-            var document = new HtmlDocument();
-            document.LoadHtml(chart.GetInlineHtml());
-
-            var divNode = document.DocumentNode.SelectSingleNode("//div");
-            var scriptNode = document.DocumentNode.SelectSingleNode("//script");
-
-            var newHtmlDocument = new HtmlDocument();
-
-            newHtmlDocument.DocumentNode.ChildNodes.Add(divNode);
-            newHtmlDocument.DocumentNode.ChildNodes.Add(GetScriptNodeWithRequire(scriptNode));
-
-            return newHtmlDocument.DocumentNode.WriteContentTo();
+            dynamic _ = new PocketView();
```

Comment:
If you `using static Microsoft.Dotnet.Interactive.Formatting.PocketViewTags` you can use `div`, etc. as top-level method calls and not have to declare a local `PocketView`.